indexer/database/bridge_transfers.go

```diff
@@ -66,7 +68,7 @@ type BridgeTransfersView interface {
 	L1BridgeDepositsByAddress(common.Address, string, int) (*L1BridgeDepositsResponse, error)
 
 	L2BridgeWithdrawal(common.Hash) (*L2BridgeWithdrawal, error)
-	L2BridgeWithdrawalSum() (float64, error)
+	L2BridgeWithdrawalSum(filter models.WithdrawFilter) (float64, error)
```

Comment:
Lets move `WithdrawFilter` into this file so that we dont introduce a dependency on `api`. This package should be self contained